examples/Analyze_Market_Volatility_Using_Historical_Data.ipynb

```diff
@@ -0,0 +1,986 @@
+            }
+          },
+          "metadata": {},
+          "execution_count": 28
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Line Plot of Closing Prices for Each Stock:"
```

Comment:
We can also use the `openbb-charting` extension to make interactive charts with Plotly.

```python
# `data` is the original response object.
# pip install openbb-charting

df = data.to_df().pivot(columns="symbol", values="close")
data.charting.to_chart(
    data=df,
    y=df.columns.tolist(),
    title="Historical Closing Price",
    scatter_kwargs=dict(hovertemplate="<b>$%{y}</b>")
)
```

Reply:
hey! i tried using this but it was causing some problems with either my python environment or dependencies, it wasn't 
recognizing openbb-charting

Reply:
If your environment doesn't recognize the extension, it probably because it isn't installed.